Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/PaintBrushManipulator.cpp

```diff
@@ -181,8 +116,10 @@ namespace AzToolsFramework
         {
             if (mouseInteraction.m_mouseInteraction.m_mouseButtons.Left())
             {
-                const bool isFirstPaintedPoint = true;
                 m_isPainting = true;
+                PaintBrushNotificationBus::Event(m_ownerEntityComponentId, &PaintBrushNotificationBus::Events::OnPaintBegin);
+
+                const bool isFirstPaintedPoint = true;
```

Comment:
nit: It looks like `isFirstPaintedPoint` is could be moved close to its first use?

Reply:
I can't get it any closer, it's already used in the next line. ;)

```
MovePaintBrush(
                    mouseInteraction.m_mouseInteraction.m_interactionId.m_viewportId,
                    mouseInteraction.m_mouseInteraction.m_mousePick.m_screenCoordinates, isFirstPaintedPoint);
```